README_dev.md

```diff
@@ -4,7 +4,8 @@
 2. download and install [Node JS](https://nodejs.org/en/download/)
 3. install the npm packages `typescript` and `cpx` **globally**
```

Comment:
What's the point in installing `typescript` and `cpx` globally? Let's add `typescript` and `cpx` as dev dependencies instead.

Reply:
cpx had vulnerabilities for a while (edit: still has) and wasn't included in the repo therefore **(leave this as is!)** 

typescript is already included, but WAY MORE better to use when installed globally. If you don't want to have it globally installed just skippit :)

Reply:
We could use https://github.com/bcomnes/cpx2 as this is maintained and a drop-in replacement

Reply:
Ok, sounds great!

Reply:
as this is not part of this PR I'll add this in another PR.